.tests/testScript_plotting.m

```diff
@@ -229,8 +229,11 @@ zlabel sqrt(rpm)
 
 %% expected to fail: patch (6) (struct doesn't call overloaded method)
```

Comment:
See prior comment regarding lack of clarity of "expected to fail" usage.

Reply:
got it. have no better idea